src/core/PathUtil.cpp

```diff
@@ -97,76 +97,91 @@ namespace lmms::PathUtil
 		}
 	}
 
-	Base baseLookup(const QString & path)
+	Base baseLookup(const QString & input)
 	{
 		for (auto base: relativeBases)
 		{
 			QString prefix = basePrefix(base);
-			if ( path.startsWith(prefix) ) { return base; }
+			if (input.startsWith(prefix)) { return base; }
 		}
 		return Base::Absolute;
 	}
 
 
+	//VstEffect saved as a platform native path in previous versions of LMMS. (PR#8082)
+	//Enforce forward slashes for compatibility.
+	QString serializePath(const QString & input)
+	{
+		return QString(input).replace("\\", "/");
+	}
 
-
-	QString stripPrefix(const QString & path)
+	QString stripPrefix(const QString & input)
 	{
-		return path.mid( basePrefix(baseLookup(path)).length() );
+		QString path = serializePath(input);
+
+		return path.mid(basePrefix(baseLookup(path)).length());
 	}
 
-	QString cleanName(const QString& path)
+	QString cleanName(const QString & input)
 	{
+		QString path = serializePath(input);
+
 		return stripPrefix(QFileInfo(path).completeBaseName());
 	}
 
 
 
-
 	QString oldRelativeUpgrade(const QString & input)
 	{
-		if (input.isEmpty()) { return input; }
```

Comment:
This guard should come first.  There's no instance where an empty string is no longer empty as a result of serialization, so this is a wasted compute cycle.

Reply:
I would recommend changing as few logic paths as possible unless you fully understand the use-case.  This particular function is "old" and to the best of my knowledge unused and a candidate for removal.  It should probably be marked with deprecated notation to help others remove it in the future.